diff --git a/node_modules/chalk/index.js b/node_modules/chalk/index.js
index 1cc5fa8..d15fdbb 100644
--- a/node_modules/chalk/index.js
+++ b/node_modules/chalk/index.js
@@ -20,8 +20,13 @@ function applyOptions(obj, options) {
 
 	// Detect level if not set manually
 	const scLevel = stdoutColor ? stdoutColor.level : 0;
-	obj.level = options.level === undefined ? scLevel : options.level;
-	obj.enabled = 'enabled' in options ? options.enabled : obj.level > 0;
+	if (typeof window !== 'undefined') {
+		obj.level = 3
+		obj.enabled = true
+	} else {
+		obj.level = options.level === undefined ? scLevel : options.level;
+		obj.enabled = 'enabled' in options ? options.enabled : obj.level > 0;
+	}
 }
 
 function Chalk(options) {
