diff --git a/node_modules/ink/build/components/App.js b/node_modules/ink/build/components/App.js
index 7719507..9889c13 100644
--- a/node_modules/ink/build/components/App.js
+++ b/node_modules/ink/build/components/App.js
@@ -5,7 +5,7 @@ Object.defineProperty(exports, "__esModule", {
 });
 exports.default = void 0;
 
-var _readline = _interopRequireDefault(require("readline"));
+// var _readline = _interopRequireDefault(require("readline"));
 
 var _react = _interopRequireWildcard(require("react"));
 
@@ -60,7 +60,7 @@ class App extends _react.PureComponent {
           stdin.resume();
           stdin.setRawMode(true);
 
-          _readline.default.emitKeypressEvents(stdin);
+          // _readline.default.emitKeypressEvents(stdin);
         }
 
         this.rawModeEnabledCount++;
